src/LostAndFound/GameManager.cs

```diff
@@ -16,5 +20,37 @@ private static void PrintIntro()
 
             TimedEvents.TimedWrite(introText, TimedEvents.WriteSpeed.Slow);
         }
+
+        private static void ManageGameActions()
+        {
+            PrintMainMenu();
+            char userInput = GetUserInput();
+        }
+
+        private static void PrintMainMenu()
+        {
+            var menuText =
+                "\n\nMain Menu:\n\t" +
+                "Press 'S' to Start a New Game\n\t" +
+                "Press 'C' to Continue a Previous Game\n\t" +
+                "Press 'L' to View the Leaderboards\n\t" +
+                "Press 'H' for Help and FAQ\n\t" +
+                "Press 'Q' to Quit\n";
+
+            TimedEvents.TimedWrite(menuText, TimedEvents.WriteSpeed.Fast);
+        }
+
+        private static char GetUserInput()
```

Comment:
Since really we are returning a user's menu choice, I wonder if we shouldn't return an enum.

Reply:
That is a good idea. I will look into it.